package.json

```diff
@@ -19,15 +19,16 @@
   "dependencies": {
     "bignumber.js": "^5.0.0",
     "bluebird": "^3.5.1",
-    "bs58": "^4.0.1",
     "ethjs-abi": "^0.2.1",
     "lodash": "^4.17.4",
     "node-fetch": "^1.7.3",
     "utf8": "^2.1.2",
     "web3-utils": "^1.0.0-beta.26"
   },
   "devDependencies": {
+    "bs58": "^4.0.1",
```

Comment:
keep this as a regular dependency. devDependencies: "is used to save the package for development purpose. Example: unit tests, minification.." Since bs58 is actually used as part of our decoder in production purposes.

Reply:
Thank you very much for your corrections and clarifications. My intention is to start working on this repository and adjust to your criteria.

I apologize in particular for this error that could break the code. It is my first contribution and I hope it does not happen again. I will correct it in the modification of this PR.

Reply:
👍